fix typos in chapter 5 (#38)
soupi/learn-haskell-blog-generator: src/05-glue/03-project.md

a subset of Hackage packages that are known to work together, or even
from remote git repositories. Usually Haskellers use a **package manager** to
download and manage packages for different projects. The most popular package

> managers are [cabal](https://cabal.readthedocs.io) and

Reviewer: These package managers are specific for Haskell
```suggestion
managers for Haskell are [cabal](https://cabal.readthedocs.io) and
```

- Where the source directory for this application is
- Which file is the 'Main' file
- Import our library, which is named `hs-blog`

> - Add additional flag for GHC, e.g., `-O` to compile with optimizations

Reviewer: ```suggestion
- Additional flags for GHC, e.g., `-O` to compile with optimizations
```

and updates the local package index which includes various information about available packages such as
their names, versions and dependencies.

> Usually the first command to run before fetching package dependencies is

Reviewer: ```suggestion
`cabal v2-update` is usually the first command to run before fetching package dependencies.
```
